packages/desktop-client/src/components/common/Toggle.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+import { css } from 'glamor';
+
+import { theme, type CSSProperties } from '../../style';
+
+type ToggleProps = {
+  id: string;
+  checked: boolean;
+  onChange?: (any) => void;
+  onColor?: string;
+  style?: CSSProperties;
+};
+
+export const Toggle = ({
+  id,
+  checked,
+  onChange,
```

Comment:
nit: Can we rename this to `onToggle`?

Reply:
I think the naming standard we are using is that only .ts files start with lowercase characters.

changing to "OnToggle" would not really follow any of the other components in the directory. If we are going to change this we should probably change them all (eg. "OnButton"), which should be a different PR. I'd suggest we keep with current standards though.

Reply:
What I meant is the onChange prop. But it's also fine to leave it as is. :)

Reply:
Ahh, sorry. Got it changed. Cheers!